### PR TITLE
Feature user add favorites clubs

### LINKED ISSUE #19: Regular user can add clubs to their list of favorite clubs
Description: 
**As a** regular user, 
**I can** mark a club as one of my favorites to my “favorite clubs” list, 
**so that** I can enter to them easily next time.
<div display="flex" align="center">
Home
Favorites
</div>

### PR DESCRIPTION
**Pull request by condef5**
### Status
Required

### Related Issues
closes #19, #20 

### Implementation Notes
* User can add clubs to their list of favorite clubs and remove clubs from list of favorite clubs
* Add module `jest-fetch-mock` for setup test asynchronous
* Add test `favorites_controller` for the backend.
* Add test view `home` for the client.

### CODE REVIEW

File: api/db/seeds.rb [hunk 1]
```diff
@@ -1,9 +1,10 @@
 puts "Init seed"
 User.destroy_all
-User.create(name: 'Lian Nivin', email: 'liam@kampu.pe', role: "regular", password: '123456')
-User.create(name: 'Cristian Berly', email: 'berli@kampu.pe', role: "owner", password: '123456')
+regular_user = User.create(name: 'Lian Nivin', email: 'liam@kampu.pe', role: "regular", password: '123456')
+p
```
Comment (cristianbgp) [inline api/db/seeds.rb]: Here's a typo :point_up:

File: client/src/components/icons.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      height="24"
+      viewBox="0 0 24 24"
+      fill="none"
+      stroke="currentColor"
+      strokeWidth="2"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+      {...props}
+    >
+      <path d="M20.84 4.61a5.5 5.5 0 0 0-7.78 0L12 5.67l-1.06-1.06a5.5 5.5 0 0 0-7.78 7.78l1.06 1.06L12 21.23l7.78-7.78 1.06-1.06a5.5 5.5 0 0 0 0-7.78z" />
```
Comment (liannivin) [inline client/src/components/icons.js]: What is that?

Comment (condef5) [reply client/src/components/icons.js]: That is an SVG, you can read more [here](https://css-tricks.com/lodge/svg/)

File: api/db/seeds.rb (hunk 1 above)
Comment (condef5) [reply api/db/seeds.rb]: Thanks `:)`

**Pull request merged**